turbinia/turbiniactl.py

```diff
@@ -388,6 +389,20 @@ def main():
       'Additionaly, you can use the -a or --all_fields flag to retrieve the '
       'full output containing finished and unassigned worker tasks.')
 
+  # Add GCS logs collector
+  parser_gcs_logs = subparsers.add_parser(
+      'DumpGCSLogs', help='Get Turbinia results from Google Cloud Storage.')
```

Comment:
Can we make the parser name lower case to match the other parsers?

Reply:
Actually, maybe we can get rid of `logs` in the name of the parser/command too since it's not just logs that we're grabbing from GCS.